enableSecurity.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+export SECURITY_SERVICE_NEEDED=true
```

Comment:
the env var is set by the jenkins job, I'm not sure we need it here?

Reply:
This script is not being called anywhere currently and it will be used for manual testing. E.g, I need to run this script, then /bin/env.sh, then setupSecurityservice.sh etc to check where might go wrong for degbugging.

Reply:
Ok that makes sense.